nodejs/src/apis/SessionService.ts

```diff
@@ -0,0 +1,74 @@
+import { APIService } from "../services/APIService";
+import { QueryResultsIterator } from "../utils/QueryResultsIterator";
+import { HMSSession } from "./interfaces/sessionInterfaces";
+
+export class SessionService {
+  private basePath = "/sessions";
+
+  constructor(private apiService: APIService) {}
+
+  /**
+   *
+   * @returns a `QueryResultsIterator<HMSSession>` object
+   */
+  getAllSessionsIterable(): QueryResultsIterator<HMSSession> {
```

Comment:
let's not add iterable in the suffix. the return type and type checks are enough hints